CHANGELOG.md

```diff
@@ -1,5 +1,67 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2025-04-27
+
+Monitor the release status by region at [AKS-Release-Tracker](https://releases.aks.azure.com/). This release is titled `v20250427`.
+
+### Announcements
+* AKS supported Kubernetes version release updates are now available in [AKS Release Tracker](https://releases.aks.azure.com/). You can check current in-support Kubernetes versions and LTS versions for specific region and track new patches version release progress with Release Tracker.
```

Comment:
Az Linux 2.0 is being retired end of November 2025, coinciding with end of support for 1.31 community version. Customers using Az Linux 2.0 are requested to migrate to 3.0 by summer when AKS shall provide a specific feature to do the migration. However there wont be any rollback capability available with this.

Reply:
@allyford feel free to edit the announcement above or review it.

Reply:
I will incorporate as Kaarthik suggested for now